skunkbooth/main.py

```diff
@@ -25,21 +26,22 @@
     level=logging.INFO,
     format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s'
 )
-vid = VideoIO()
 
 
 def main() -> None:
     """Main driver function"""
+    vidBuf = Queue(99999)
```

Comment:
Make it 32767. Unfortunately macOS has constrain on this value.
Usually it is same as INT_MAX but, macOS being macOS SEM_VALUE_MAX is 32767.

```shell
❯ python3 -c "import _multiprocessing; import platform; print (platform.platform(),  _multiprocessing.SemLock.SEM_VALUE_MAX)"
macOS-10.16-x86_64-i386-64bit 32767
```